src/renderer/components/+namespaces/namespace-select-filter.tsx

```diff
@@ -57,6 +57,8 @@ const Placeholder = observer((props: PlaceholderProps<any>) => {
 
 @observer
 export class NamespaceSelectFilter extends React.Component<SelectProps> {
+  static isMulti = false;
```

Comment:
This should be observable
```suggestion
  static isMulti = observable.box(false);
```